packages/gestalt/src/ExpandableModuleBase.js

```diff
@@ -0,0 +1,108 @@
+// @flow strict
+import React, { type Node } from 'react';
+import Box from './Box.js';
+import IconButton from './IconButton.js';
+import Icon from './Icon.js';
+import TapArea from './TapArea.js';
+import Text from './Text.js';
+
+type Props = {|
+  title: string,
+  icon?: string,
+  iconAccessibilityLabel?: string,
+  summary?: Array<string>,
+  isCollapsed: boolean,
+  onModuleClicked?: boolean => void,
+  type?: string,
+  children?: Node,
+|};
+
+const EXPANDABLE_TYPE_ATTRIBUTES = {
```

Comment:
nit: Can rename to something shorter like "EXPANDABLE_TYPES"

Reply:
I think EXPANDABLE_TYPE_ATTRIBUTES is more accurate.